sympy/solvers/pde.py

```diff
@@ -230,7 +230,8 @@ def _handle_Integral(expr, func, order, hint):
         return expr
 
 
-def classify_pde(eq, func=None, dict=False, **kwargs):
+def classify_pde(eq, func=None, dict=False, ics=None, xi=None, eta=0,
+        n=None, x0=0, prep=True):
```

Comment:
Are these other arguments used?

Reply:
From `dsolve()`, it is called with all these parameters. They are not seems to be used but they are getting passed by some other functions.

Reply:
If they're not used they should be removed. The only case where you would have unused parameters is if the function needs to have a specific signature to follow some API, which is not the case here.

Reply:
Should I refactor it for rest of the  functions who are using it?

Reply:
Yes